common/http/platform/beast/http.cpp

```diff
@@ -98,8 +97,6 @@ Client::Client(ClientConfig &client, events::EventLoop &event_loop) :
 	cancelled_(make_shared<bool>(false)),
 	resolver_(GetAsioIoContext(event_loop)),
 	body_buffer_(HTTP_BEAST_BUFFER_SIZE) {
-	response_buffer_.reserve(body_buffer_.size());
```

Comment:
@oleorhagen: I was wrong. This is safe, looks like the buffer is automatically resized on demand.

Reply:
However, it does degrade performance significantly, because it uses the smallest buffer size available. I dug through the source code and found a way to reserve using a different approach instead.

Reply:
What did you do?

Reply:
I just looked at what `reserve` itself does. 🙂